app/routers/api/notifiers.py

```
@@ -76,9 +79,11 @@ def create_notifier(notifier: NotifierCreate, db: Session = Depends(get_db)):
     # Check for duplicate name
     existing = db.query(NotifierModel).filter(NotifierModel.name == notifier.name).first()
     if existing:
+        # Sanitize error message: Do not reflect user input 'name'
+        logger.info(f"Attempt to create duplicate notifier: {sanitize_for_log(notifier.name)}")
```

Ruff (0.15.0)

[warning] Logging statement uses f-string

(G004)